internal/helper/helper.go

```diff
@@ -298,6 +302,21 @@ func SetCommandOutput(cmd *exec.Cmd, liveOutput bool) (cmdOutput *bytes.Buffer)
 	return cmdOutput
 }
 
+// RunCmds runs a list of commands and returns the error
+// It stops at the first error
+func RunCmds(cmds []*exec.Cmd, debug bool) error {
```

Comment:
This function is indirectly tested because callers are tested but since they are in another package it seems the coverage is not registered. I am not sure it is worth writing dedicated unit tests for it.

Reply:
There is no way in Go to add a comment in the code to ignore a function/block in the coverage (see https://github.com/golang/go/issues/53271 which was rejected). The only way I know to "hide" the fact that some lines miss coverage is to go do some surgery in the resulting coverprofile. I am not in favor of doing so because I think we will forget about it and this may lead to weird cover report. I do not think this is a big deal that our cover report and percentage are a bit "off". This is an imprecise metric anyway.